app/models/miq_server/worker_management/base.rb

```diff
@@ -0,0 +1,45 @@
+class MiqServer::WorkerManagement::Base < MiqServer::WorkerManagement
```

Comment:
Can `MiqServer::WorkerManagement` just _be_ the base, without having an extra layer called "Base"?

Thinking `class MiqServer::WorkerManagement::Kubernetes < MiqServer::WorkerManagement`

Reply:
:+1: done